### ISSUE TITLE
Request: Filters on carousels

### ISSUE THREAD

**Issue by dekatron** [gndev/shortcodes-ultimate#86]
Hi,

Would it be possible to add a couple of filters in the carousels shortcode? I don't mind doing it myself as a pull request if you're happy with the filters / naming conventions but I'm thinking something like:

line 167, /includes/shortcodes/carousel.php
```
$atts = apply_filters( 'su/carousel_args', $atts );
```

line 186, /includes/shortcodes/carousel.php
```
$slides = apply_filters( 'su/carousel_slides', $slides );
```

**Comment by gndev** [comment]
Hi @dekatron,

as for `su/carousel_args`, you can use the `shortcode_atts_{$shortcode}` filter. [Codex](https://developer.wordpress.org/reference/hooks/shortcode_atts_shortcode/).

Example:

```php
add_filter( 'shortcode_atts_carousel', function( $out, $pairs, $atts, $shortcode ) {
  return $out;
}, 10, 4 );
```

For `su/carousel_slides` I'll add something similar in a few days.

**Comment by dekatron** [comment]
Ah! thanks, I didn't realise that filter existed!

**Comment by gndev** [comment]
Hi @dekatron,

just added new filters:

1. `su/shortcode/slider/slides`
2. `su/shortcode/carousel/slides`
3. `su/shortcode/custom_gallery/slides`

New filters will be available with v5.0.8.

**Comment by dekatron** [comment]
Thanks, very much appreciated :)
